src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/SqrtT_ConsistentOneColumnTableResult.txt

```diff
@@ -11,3 +11,18 @@ Errors: Error 0-8: The function 'Sqrt' has some invalid arguments.|Error 5-7: In
 
 >> Sqrt([Blank(), 1444])
 [Blank(),38]
+
+>> Sqrt(If(1<0,[1]))
+Blank()
+
+>> Sqrt(Table({a:16}, {a:289}, {a:1/0}))
+[4,17,Microsoft.PowerFx.Types.ErrorValue]
+
+>> Sqrt(Table({a:6.25}, {a:Power(2,2)}, {a:-9}))
+[2.5,2,NaN]
```

Comment:
>NaN [](http://example.com/codeflow?start=7&length=3)

Similar comment to the Abs/Exp tests: this should be an error value. We should not have non-finite values (Infinity, NaN) as the result of Power Fx expressions

Reply:
Removed invalid cases from this PR